libs/common/parameters_utils.h

```diff
@@ -154,6 +154,8 @@ static inline bool VtValueGetBool(const VtValue& value, bool defaultValue = fals
         return value.UncheckedGet<bool>();
     if (value.IsHolding<int>())
         return value.UncheckedGet<int>() != 0;
+    if (value.IsHolding<unsigned int>())
```

Comment:
yes, the only problem I saw is that the docs seem to say that this is not thread-safe if the same VtValue is used in parallel. I'm not sure if this could happen on our side